tests/tests_geomstats/test_full_rank_correlation_matrices.py

```diff
@@ -75,7 +75,7 @@ def test_vertical_projection_is_vertical(self, n, tangent_vec, mat):
         result = gs.all(bundle.is_vertical(proj, gs.array(mat)))
         self.assertTrue(result)
 
-    @autograd_tf_and_torch_only
+    @autograd_and_torch_only
```

Comment:
There is an inconsistency in the imports, sometimes we say:

`@tests.conftest.np_and_autograd_only`

and sometimes only `@autograd_and_torch_only`.

What do you recommend and do you think this could be made consistent?

Reply:
It really is a matter of taste here. My personal preference is to keep the whole line. You can easily do this with a blend of find and sed.

something like:

```
find . -iname "*.py" -exec sed -i s/@autograd_and_torch_only/@tests.conftest.np_and_autograd_only/g {} \;
```

Just be consistent.